src/extension/src/constants.ts

```diff
@@ -394,3 +395,8 @@ export enum OS {
   Linux = "linux",
   Windows = "windows"
 }
+
+export const BackendFrameworkLinuxVersion: { [s: string]: string } = {
+  Node: "node|lts",
+  Flask: "python|3.7"
```

Comment:
What are the versions used for? Do they mean that the app service would look for code that is compatible with this version of python?

Reply:
version is for the runtime stack of the web app instance created